account/views.py

```diff
@@ -40,3 +40,28 @@ def post(self, request):
         except Exception as e:
             return Response({'status': False, 'message': str(e)},
                             status=status.HTTP_400_BAD_REQUEST)
+
+class GetStartTime(APIView):
+    def get(self, request):
+        try:
+            user = request.user
```

Comment:
so the reason you are getting Anonymous User maybe you didn't login correctly. Check my loom video on the previous PR for how to login with an access token.